tools/clang/test/CodeGenSPIRV/decoration.user-type.hlsl

```diff
@@ -80,6 +80,12 @@ cbuffer MyCBuffer { float x; };
 // CHECK: OpDecorateString %MyTBuffer UserTypeGOOGLE "tbuffer"
 tbuffer MyTBuffer { float y; };
 
+// CHECK: OpDecorateString %bab UserTypeGOOGLE "byteaddressbuffer"
```

Comment:
Just a nit. Can't we make it as case-sensitive?

Reply:
The spec for UserTypeGoogle decoration specifies that the type should be lower-case